metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -247,10 +251,14 @@ def build_service_principal_map(self) -> None:
 
     def process_notebooks(self) -> Iterable[MetadataWorkUnit]:
         for notebook in self.unity_catalog_api_proxy.workspace_notebooks():
+            if not self.config.notebook_pattern.allowed(notebook.path):
```

Comment:
was this just missing before?

Reply:
I added the pattern as part of this PR, forgot it last time